plugin/innodb_memcached/innodb_memcache/src/innodb_engine.c

```diff
@@ -1135,22 +1135,22 @@ innodb_conn_init(
 			}
 		}
 	} else {
+        bool auto_commit = (engine->read_batch_size == 1 &&
+                !(engine->cfg_status & IB_CFG_DISABLE_ROWLOCK)) ? true : false;
```

Comment:
When will this "bool foo = condition ? true : false" stop
(does not need fixing here)

Reply:
Yeah :)
This is not the best example though.
My favorite is
```c++
if(flag == true)
{
  ...
}
```